src/zeit/content/cp/testing.py

```diff
@@ -38,14 +40,36 @@
 """.format(fixtures=pkg_resources.resource_filename(__name__, '.'))
 
 
-ZCML_LAYER = zeit.cms.testing.ZCMLLayer(
-    'ftesting.zcml',
+CP_LAYER = zeit.cms.testing.ZCMLLayer(
+    'ftesting.zcml', name='CPLayer',
     product_config=zeit.cms.testing.cms_product_config
     + zeit.workflow.testing.product_config
     + zeit.content.image.testing.product_config
     + product_config)
 
 
+# XXX copy&paste from zeit.solr.testing, because importing that would create an
+# import cycle with zeit.content.article.testing, sigh.
```

Comment:
Könnte man den Layer von `zeit.solr.testing` in der `__init__` aufbauen? In `zeit.push` wird so ein Import Cycle von der `product_config` umgangen.